presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -4774,6 +4774,238 @@ public void testPageFileCompression()
+                ", _date DATE" +
+                ", _timestamp TIMESTAMP" +
+                ")" +
+                "WITH (format = 'parquet')";
+
+        Session session = Session.builder(getSession())
+                .setCatalogSessionProperty(catalog, "enable_partial_aggregation_pushdown", "true")
+                .setCatalogSessionProperty(catalog, "enable_partial_aggregation_pushdown_for_variable_length_datatypes", "true")
+                .build();
+        assertUpdate(session, createTable);
+
+        TableMetadata tableMetadata = getTableMetadata(catalog, TPCH_SCHEMA, "test_parquet_table");
+        assertEquals(tableMetadata.getMetadata().getProperties().get(STORAGE_FORMAT_PROPERTY), HiveStorageFormat.PARQUET);
+
+        assertUpdate(session, "INSERT INTO test_parquet_table VALUES (" +
+                "true" +
+                ", cast(1 as tinyint)" +
+                ", cast(2 as smallint)" +
+                ", 3" +
+                ", 4" +
+                ", 1.2" +
+                ", 2.3" +
+                ", 4.5" +
+                ", 55555555555555.32" +
+                ", 'abc'" +
+                ", 'def'" +
+                ", 'g'" +
+                ", 'hij'" +
+                ", cast('klm' as varbinary)" +
+                ", cast('2020-05-01' as date)" +
+                ", cast('2020-06-04 16:55:40.777' as timestamp)" +
+                ")", 1);
+
+        assertUpdate(session, "INSERT INTO test_parquet_table VALUES (" +
+                "false" +
+                ", cast(10 as tinyint)" +
+                ", cast(20 as smallint)" +
+                ", 30" +
+                ", 40" +
+                ", 10.25" +
+                ", 25.334" +
+                ", 465.523" +
+                ", 88888888555555.91" +
+                ", 'foo'" +
+                ", 'bar'" +
+                ", 'b'" +
+                ", 'baz'" +
+                ", cast('qux' as varbinary)" +
+                ", cast('2020-06-02' as date)" +
+                ", cast('2020-05-01 18:34:23.88' as timestamp)" +
+                ")", 1);
+        String rowCount = "SELECT 2";
+
+        assertQuery(session, "SELECT COUNT(*) FROM test_parquet_table", rowCount);
+        assertQuery(session, "SELECT COUNT(_boolean) FROM test_parquet_table", rowCount);
+        assertQuery(session, "SELECT COUNT(_tinyint) FROM test_parquet_table", rowCount);
+        assertQuery(session, "SELECT COUNT(_smallint) FROM test_parquet_table", rowCount);
+        assertQuery(session, "SELECT COUNT(_integer) FROM test_parquet_table", rowCount);
+        assertQuery(session, "SELECT COUNT(_bigint) FROM test_parquet_table", rowCount);
+        assertQuery(session, "SELECT COUNT(_real) FROM test_parquet_table", rowCount);
+        assertQuery(session, "SELECT COUNT(_double) FROM test_parquet_table", rowCount);
+        // The default Parquet writer does not populate statistics for these datatypes
+        // though they are populated by hive etl tasks
+        /*
```

Comment:
I'm not sure if I followed...Should we actually remove these tests that are commented out?

Reply:
I'll enabled these. This work was initially developed on an older version of Presto where these statistics were not properly populated. This is fixed on master